File: levels/0/get_secret.js
const {SecretManagerServiceClient} = require('@google-cloud/secret-manager');
const client = new SecretManagerServiceClient();
async function accessSecretVersion(name) {
  const [version] = await client.accessSecretVersion({
    name: name,
  });

  // Extract the payload as a string.
  const payload = version.payload.data.toString('utf8');
  return payload
}

accessSecretVersion(process.argv[2]).then(
  console.log
)

File: levels/3/get_secret.py
import sys
import argparse
from google.cloud import secretmanager

def get_secret(name):
    client = secretmanager.SecretManagerServiceClient()
    response = client.access_secret_version(name)
    payload = response.payload.data.decode('UTF-8')
    sys.stdout.write(payload)

if __name__ == "__main__":
    get_secret(sys.argv[1])
